capstone/src/main/resources/static/style.css

```diff
@@ -0,0 +1,122 @@
+#content {
```

Comment:
Let me know when you decide to merge and we can hop on a call together. I also started a style.css and we probably would have lot of conflicts.

Reply:
Btw, you can have multiple style sheets.
You can have an `event.css` that has styles specific to the event page, and also `style.css` that applies to general aspects of all pages.
Both style sheets can be included on the same HTML page.
This might alleviate the potential for merge conflicts.

Reply:
I'll change the file name on my end